content/app/commands/highlight-text-command.js

```diff
@@ -0,0 +1,111 @@
+define("content/app/commands/highlight-text-command", function(require, exports, module) {
+
+	// imports
+	var Event =require("content/app/events/event");
+	var AbstractCommand = require("content/app/commands/abstract-command");
+	var CommitPageHelper = require("content/app/github/commit-page-helper");
+	var WindowHelper = require("content/app/github/window-helper");
+	var Box = require("content/app/models/box");
+
+	/**
+	 * This command is used for selecting text in files.
+	 */
+	var HighlightTextCommand = Class(AbstractCommand, {
+
+		/**
+		 * HighlightTextCommand constructor.
+		 */
+		constructor : function () {
+			HighlightTextCommand.$super.call(this);
+
+			this._onSelectHandler = this._onSelect.bind(this);
+			this._highlights = [];
```

Comment:
highlight nodes it general too... proper name would be highlightDOMElements or similar. But I think that context of this class is too small that you won't know what is it about.